portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -5,11 +5,11 @@
 
   private final long id;
   private final String comment;
-  private final long timestamp;
   private final String firstname;
   private final String lastname;
+  private final long timestamp;
 
-  public Comment(long id, String comment, String lastname, String firstname, long timestamp) {
+  public Comment(long id, String comment, String firstname, String lastname, long timestamp) {
```

Comment:
In Java, identifiers are typically camel-case, meaning they start with a lowercase letter but each word after that the first letter is uppercase. So, firstName and lastName in this case.